### DISCUSSION TITLE
não consigo usar PO em um módulo diferente do principal

### DISCUSSION THREAD

**Discussion by viniciusgati** [po-ui/po-angular#1691] [Angular] [answered]
Olá boa tarde, 

estou tentando usar o PO em um modulo novo que criamos porém neste module mesmo importanto tudo certo o angular não encontra os componentes do angular...

Existe algum esquema para fazer isso funcionar?

**Comment by anliben** [comment]
Bom Dia @viniciusgati, poderia fazer um exemplo utilizando o stackblitz? Só para eu entender melhor: https://stackblitz.com/edit/po-ui


Ator.

**Reply by viniciusgati** [reply]
https://stackblitz.com/edit/angular-uydihy
Taí link do stackblitz

**Reply by anliben** [reply]
Ola @viniciusgati, vi que no seu código está usando um componente dentro do outro e isso faz com que o modulo do po-ui ou outros módulos não funcione, pois ele não reconhece o componente em sua raiz, isso e um problema no código, a lib esta correta, a interessante séria ver sobre declaração de módulos no angular e componentes em mais de um nível!

**Reply by viniciusgati** [reply]
Certo mas no caso, eu não posso ter ou tem alguma configuração que eu consiga usar assim?

**Comment by viniciusgati** [comment, marked as answer]
Pra quem tiver o mesmo problema a minha configuraćão estava errada, eu estava no AppModule dando load somente no modulo de rotas do outro modulo, assim que coloquei o modulo em si no imports do appmodule já funcionou corretamente.
